app/src/server.js

```diff
@@ -1,15 +1,22 @@
 const path = require('path')
 const express = require('express')
+const query = require('./queries.js')
 const app = express()
 
 const port = process.env.port || 8111
 
 const staticFilesPath = path.join(__dirname, '../public')
 app.use(express.static(staticFilesPath))
 
-// app.get('/', function(req,res){
-//   res.send('MARIOOOOOOOO pshu pshu pshu')
-// })
+app.get('/all-nuggets', function (req, res) {
+  query.getAll((err, result) => {
+    if (err) {
+      console.log(err)
+      res.send('DATABSE ERROR')
```

Comment:
I think you need a return inside this block, otherwise there will be an error (maybe)
i.e. return res.send(...